include/ginkgo/core/matrix/ell.hpp

```diff
@@ -85,6 +85,7 @@ class Ell : public EnableLinOp<Ell<ValueType, IndexType>>,
     friend class Ell<to_complex<ValueType>, IndexType>;
 
 public:
+    using ReadableFromMatrixData<ValueType, IndexType>::read;
     using EnableLinOp<Ell>::convert_to;
     using EnableLinOp<Ell>::move_to;
```

Comment:
nit: Previously, you placed it below `move_to` and `convert_to`:
```suggestion
    using EnableLinOp<Ell>::convert_to;
    using EnableLinOp<Ell>::move_to;
    using ReadableFromMatrixData<ValueType, IndexType>::read;
```